### ISSUE TITLE
Eyebrows not working

### ISSUE THREAD

**Issue by CoderKing06** [IthanMendoza/Fresh-Moves#177]
![SkinMC-d638461d-1658-4858-b447-e3ddc3474494](https://github.com/user-attachments/assets/bd69e47d-9022-4d55-8784-fe17c09d0604)
I am trying to use the upper mid eyebrows. The eyes are working but the eyebrows don't seem to be moving. Is there a problem with my skin? Do I need to move something?

**Comment by CoderKing06** [comment]
If this is important, I've only tested it in Multiplayer so far.

**Comment by CoderKing06** [comment]
I looked at one of the example skins and this might have fixed my issue, although if this is the solution then the wiki was incredibly unhelpful.
![SkinMC-d638461d-1658-4858-b447-e3ddc3474494-1 png](https://github.com/user-attachments/assets/8349eefc-73b3-4799-9d20-02ae854a2037)

**Comment by CoderKing06** [comment]
Ok it didn't work someone smart please help me

**Comment by SaltSouls** [comment]
Yeah, I've also been trying to work out the upper mid eyebrows. Nothing I do seems to make them work. In fact, it seems that just all brows I've tested so far don't work.

**Comment by CoderKing06** [comment]
I tried one of the example skins and even that didn't work, maybe it's just broken right now.

**Comment by idkjustollie** [comment]
your eyebrows were in the wrong spot for the first one
![364876753-8349eefc-73b3-4799-9d20-02ae854a2037](https://github.com/user-attachments/assets/c8bb1ab2-4462-48f5-a568-e42f75d79f07)
